rpcs3/Emu/RSX/VK/VKGSRender.cpp

```diff
@@ -2400,14 +2400,27 @@ void VKGSRender::patch_transform_constants(rsx::context* ctx, u32 index, u32 cou
 		data_source = iobuf.data();
 	}
 
+	// Preserving an active renderpass across a transfer operation is illegal vulkan. However, splitting up the CB into thousands of renderpasses incurs an overhead.
+	// We cheat here for specific cases where we already know the driver can let us get away with this.
+	static const rsx::simple_array<vk::driver_vendor> s_allowed_vendors =
```

Comment:
Would a std::set be slower?

Reply:
We're not really searching a tree so it should perform the same. There is no order to the elements provided.
It doesn't really matter either way for the moment, this PR is likely to do nothing and would just be closed if it doesn't work.